src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K,V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
initialize table through the constructor, not directly. it allows flexibility to choose a different initial capacity or modify the initialization logic if needed. 
public MyHashMap() {
    table = new Node[DEFAULT_INITIAL_CAPACITY];
}